_rules/meta-viewport-b4f0c3.md

```diff
@@ -0,0 +1,135 @@
+---
+id: b4f0c3
+name: meta viewport
+rule_type: atomic
+description: |
+  This rule checks that the meta element is not used to block the user agent ability to zoom. 
+accessibility_requirements:
+  wcag20:1.4.10: # Reflow (AA)
```

Comment:
I don't think this is a failure of 1.4.10. Instead I would say this is a failure of 1.4.4, provided you allow max scale of 400% or greater. My reasoning is this, SC 1.4.10 says that when content is presented in this 320px width / height, it doesn't require two-dimensional scrolling. It doesn't actually say that it always has to be possible to display content in this 320px width / height. It certainly implies it, but I would argue that the reason it implies it is because SC 1.4.4 requires it. If you read the intent in the understanding document, it even mentions that 400% zoom. https://www.w3.org/WAI/WCAG21/Understanding/reflow.html